includes/components/class-attribute.php

```diff
@@ -1560,6 +1560,21 @@ public function add_meta_boxes( $meta_boxes ) {
 
 					if ( 'attributes' === $meta_box_name ) {
 						$meta_box['screen'] = $model;
+
+						if ( 'listing' === $model ) {
```

Comment:
Check the comment below regarding adding support for all the models that support categories (e.g. we also have this for vendors, bookings, requests). Once you do this please check if you can move the category field above in the array of default meta box fields, but I don't remember if all the models (e.g. user) have categories

Reply:
Only post-based models have categories

Reply:
Besides users, bookings also don't have their own categories, so moving this in the default meta box fields is not possible